libsolidity/ast/Types.h

```diff
@@ -132,7 +132,7 @@ class Type: private boost::noncopyable, public std::enable_shared_from_this<Type
 public:
 	enum class Category
 	{
-		Integer, IntegerConstant, StringLiteral, Bool, Real, Array,
+		Integer, NumberConstant, StringLiteral, Bool, Fixed, Array,
```

Comment:
I think we should call it `FixedPoint`, otherwise it might be mistaken for `FixedBytes`.